tests/migrations/test_migrations_simple_atomic_migration/0001_initial.py

```diff
@@ -0,0 +1,21 @@
+from django.db import migrations, models
+
+
+def test_object(apps, schema_editor):
+    # Test atomic operation in non-atomic migration is wrapped in transaction
+    Author = apps.get_model('migrations', 'Author')
+    Author.objects.create(name='Test Author')
```

Comment:
There's no point in performing anything here as it's not tested. Use `RunPython.noop` below.